ui/ui-frontend/projects/archive-search/src/app/archive/archive-search/additional-actions-search/reclassification/reclassification.component.html

```diff
@@ -44,11 +45,31 @@
         </div>
 
         <div *ngIf="precided || 1000 > totalChilds">
-          <span *ngIf="!pendingGetChilds" class="badge badge-secondary pad"
-            >{{ totalChilds }} {{ 'RECLASSIFICATION.FIRST_STEP.CHILDS' | translate }}</span
+          <span *ngIf="!pendingGetChilds" class="badge badge-secondary pad">
+            {{ 'RECLASSIFICATION.FIRST_STEP.INCLUDING_NB_FOLDERS_DOCUMENTS' | translate: { nbDocuments: totalChilds } }}</span
           >
         </div>
 
+        <div class="text medium mb-2">
+          @if (archiveUnitFetchedParents.length >= 1 && archiveUnitFetchedParents.length <= 5) {
+            {{
+              archiveUnitFetchedParents.length
+                | i18nPlural: currentParentFolderMap
+                | translate: { nbParents: archiveUnitFetchedParents.length }
+            }}
+          } @else if (archiveUnitFetchedParents.length > 5) {
+            {{ 'RECLASSIFICATION.FIRST_STEP.MORE_THAN_FIVE_CURRENT_PARENTS_FOLDERS' | translate }}
+          } @else if (!hasParents) {
+            <div class="text-title">{{ 'RECLASSIFICATION.COMMON.ARCHIVE_WITH_NO_PARENTS' | translate }}</div>
```

Comment:
```suggestion
            <div class="light">{{ 'RECLASSIFICATION.COMMON.ARCHIVE_WITH_NO_PARENTS' | translate }}</div>
```
Mais y'a pas d'exemple dans le Figma...